Majasdarbs/06-majasdarbs/OlegsKozlovs/app.js

```diff
@@ -0,0 +1,30 @@
+const characters = ["a", "b", "c", "d", "e", "g", "k", "m", "o", "q", "t", "z"];
+
+let arrayId;
+
+let output = document.getElementById("main");
+
+const printValues = () => { output.innerText = characters[arrayId]};
+
+const getValue = () => {
+    arrayId = prompt("Please provide character index:");
+    if (!isNaN(arrayId)) {
+        if (arrayId >= 0 && arrayId <= characters.length - 1) {
+            printValues();
+            return;
+        }
+    }
+    alert("Invalid index has been provided! Please try again.");
```

Comment:
Idea: instead of showing alert together with prompt, you could pass in getValue(promptMessage)
and in prompt you could use that message. prompt(promptMessage ?? "Please provide character index:")